030-Concepts/080-pricing.mdx

```diff
@@ -7,26 +7,195 @@ slug: concepts/pricing
 
-### Request Rate limit
+## Billing terms
 
-The maximum number of requests allowed per second. If the request rate limit is exceeded additional requests will fail with a `429` HTTP status.
+### Concurrency limit
 
-This limit exists on shared Xata instances to ensure reliability for all users on the instance.
+The concurrency limit refers to the number of queries that can run simultaneously. It's the cap on how many requests we process at once. Unlike a "connections limit", this doesn't count the total connections, but only the active queries. Even if a large number of connections are made, only a certain number of them can execute queries at the same time. This protects against very slow queries and ensures that extremely time-consuming queries don't stall the entire system.
 
-### Request Concurrency limit
+To ensure optimal performance for users, we've set the following limits. These allocations are designed to prioritize and balance the workload across the different store types based on requirements and usage patterns:
 
-The maximum number of simultaneous requests that can be processed at any given time.
+  -  **Free plan:** 15 parallel requests (6 primary + 9 from replicas) to the database
+  -  **Pro plan:** 45 parallel request (18 primary + 27 from replicas) to the database
 
-If the request concurrency limit is exceeded additional requests will fail with a `429` HTTP status.
+> These allocations are 'soft' limits. If the high usage isn't accidental or abusive, Xata can adjust these caps. Xata operates on shared clusters, and aims to ensure that one user's high usage doesn't negatively affect everyone else using that cluster.
 
-These allocations are designed to prioritize and balance the workload across the different store types based on requirements and usage patterns.
+If requests surpass the concurrency limit, they'll receive a 429 HTTP status, indicating they've been rejected due to overload.
+
+Limits apply to each workspace and are distributed among the databases and branches within that workspace.
 
 ### Data Size
 
-The data size on disk denotes the quantity of data held within the transactional database.
+The disk's data size represents the amount of data stored in the transactional database.
 
 This size limit is considered a soft-limit and is not strictly enforced in real-time. However, if you exceed this limit, it indicates that your data storage requirements may be approaching a critical level. In such cases, we recommend you explore potential solutions with [Xata Support](https://support.xata.io/hc/en-us) to optimize and scale your data storage.
 
-If you have any further questions or need assistance, don't hesitate contact [Support](https://support.xata.io/hc/en-us).
+### Point-in-Time Recovery
```

Comment:
I think even if it's not self-service, I think it would be good to mention we do PITR through Support as required. It at least let's the user know that we're doing backups for them, even if it's a little awkward to request.

Without mentioning it, the user will just think... well, there are NO backups at all happening.

Reply:
If we do leave it in, this feature needs to be classified to grid. We currently list it as a Free plan feature. Right now it loosely exists on this page only as a glossary item.

Reply:
I'd leave the call to @tsg and it's his decision. For my part I'm just highlighting user perception.